frontend/src/lib/api.mock.ts

```diff
@@ -99,13 +100,15 @@ export const MOCK_DEFAULT_TEAM: TeamType = {
     capture_dead_clicks: false,
     human_friendly_comparison_periods: false,
     revenue_tracking_config: {
+        baseCurrency: CurrencyCode.USD,
         events: [
             {
                 eventName: 'purchase',
                 revenueProperty: 'value',
-                revenueCurrencyProperty: undefined,
+                revenueCurrencyProperty: { static: CurrencyCode.ZAR },
```

Comment:
Used ZAR because South Africa's flag looks cool on the snapshot and you must be tired of me adding the Brazilian flag everywhere :)

Reply:
Are we though? 🇧🇷